features/cdmf-base-ui/org.wso2.carbon.devicemgt.ui.feature/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag

```diff
@@ -109,8 +127,26 @@ if (!user) {
         } else {
             //URL: https://localhost:9443/{deviceType}/download?owner={username}
             deviceManagerService = devicemgtProps["httpsURL"] + "/" + deviceType + "/manager";
-
             sketchGenerateLinkEndPoint = deviceManagerService + "/device/" + sketchType + "/generate_link";
+
+            //check if config exists
+            var deviceTypeConfigFile = new File("../app/units/cdmf.unit.device.type."+deviceType+".type-view/private/conf/device-type.json");
+            if(deviceTypeConfigFile.isExists()){
+                deviceTypeConfigFile.open("r");
```

Comment:
Agreed. This logic has already moved to utility.js and fixed with [#01342e0](https://github.com/wso2/carbon-device-mgt/commit/01342e03f5516ca9a93ec077fe3e457a25d65137)